src/hw-monitor.h

```diff
@@ -204,7 +204,7 @@ namespace librealsense
         {}
 
         std::vector<uint8_t> send_receive(
-            const std::vector<uint8_t>& data,
+            uint8_t const * pb, size_t cb,
```

Comment:
What pb & cp means?
Don't you prefer meaningful names?

Reply:
How PB & CP are data and data_size? :)

Reply:
It's pretty common Hungarian notation for "pointer to bytes" and "count of bytes".